tasks/add_new_openwrap_partner.py

```diff
@@ -1127,81 +1279,88 @@ def main():
+        {name_start_format}bidders{format_end} = {value_start_format}{bidder_code}{format_end}
+        {name_start_format}placements{format_end} = {value_start_format}{placements}{format_end}
+        {name_start_format}custom targeting{format_end} = {value_start_format}{custom_targeting}{format_end}
+        {name_start_format}same advertiser exception{format_end} = {value_start_format}{same_adv_exception}{format_end}
+        {name_start_format}device categories{format_end} = {value_start_format}{device_categories}{format_end}
+        {name_start_format}device capabilities{format_end} = {value_start_format}{device_capabilities}{format_end}
+        {name_start_format}roadblock type{format_end} = {value_start_format}{roadblock_type}{format_end}
+        """.format(
+        num_line_items = len(prices),
+        order_name=order_name,
+        advertiser=advertiser_name,
+        advertiser_type=advertiser_type,
+        lineitem_type=lineitem_type,
+        lineitem_prefix=lineitem_prefix,
+        setup_type=setup_type,
+        user_email=user_email,
+        prices_summary=prices_summary,
+        bidder_code=bidder_code,
+        placements=placements_print,
+        sizes=sizes,
+        custom_targeting=custom_targeting,
+        same_adv_exception=same_adv_exception,
+        device_categories=device_categories,
+        device_capabilities=device_capabilities,
+        roadblock_type=roadblock_type,
+        use_1x1=use_1x1,
         name_start_format=color.BOLD,
         format_end=color.END,
         value_start_format=color.BLUE,
-    ))
+        ))
 
+    if setup_type == constant.ADPOD:
+        # Prininting adpod price based lineitem summary
+        logger.info(
+        u"""
+        ADPOD Details :
+        {name_start_format}Adpod Size{format_end}: {value_start_format}{adpod_size}{format_end}
+        {name_start_format}Adpod creative Durations{format_end}: {adpod_creative_durations}{format_end}
+        {name_start_format}Line Item per each slot{format_end}: {value_start_format}{lineitem_per_slot}{format_end}
+        {name_start_format}Creatives per each slot{format_end}: {value_start_format}{creatives_per_slot}{format_end}
+        {name_start_format}Total number of Line Items{format_end}: {value_start_format}{lineitem_total}{format_end}
+        {name_start_format}Total number of Creatives{format_end}: {value_start_format}{creatives_total}{format_end}      
+        """.format(
+            adpod_size = adpod_size,
+            lineitem_per_slot = len(prices),
+            adpod_creative_durations = adpod_creative_durations,
+            creatives_per_slot= len(adpod_creative_durations),
+            lineitem_total = len(prices)*adpod_size,
+            creatives_total= adpod_size * len(adpod_creative_durations),
+            name_start_format=color.BOLD,
+            format_end=color.END,
+            value_start_format=color.BLUE,
+        ))
+  else:
```

Comment:
In reference to the above comment, this should be executed for: 
`if deal_lineitem_enabled `

Reply:
Printing all those values as part of **print_summary** function for  Adpod deallineitems